src/services/connections.js

```diff
@@ -219,4 +219,66 @@ module.exports = class ConnectionHelper {
 			throw error
 		}
 	}
+	static async list(pageNo, pageSize, searchText, queryParams, userId, orgId) {
```

Comment:
@nevil-mathew can you from this PR going forward add function signature and comments as done - https://github.com/ELEVATE-Project/survey-project-creation-service/blob/2048c8c42370887c89d1cf629fe9847b9d5aa33c/src/services/resource.js#L986